Yoast/wordpress-seo
# Make AMP pages output Yoast SEO Schema

## Summary

This PR can be summarized in the following changelog entry:

* Yoast SEO now takes control of the Schema output on pages generated by the [AMP plugin](https://wordpress.org/plugins/amp/). 

## Relevant technical choices:

* Removed the AMP schema metadata.

## Test instructions
This PR can be tested by following these steps:

* Go to an AMP page with the AMP plugin active, see that we have nice Yoast SEO schema.

## Quality assurance


Fixes #12862
